README.md

```diff
@@ -78,6 +78,24 @@ kubectl delete pvc <PVC ids here>
 
 Or delete the related kubernetes namespace, which contains the resources.
 
+## Deprecation of zeebe-*-helm
+
+With the creation of the Camunda Cloud Self Managed (CCSM) helm charts, the old zeebe-* charts have been deprecated.
+This means they are no longer part of the repository and no longer maintained. The packaged charts are still available
+for download.
+
+The following charts are deprecated:
+
+ * zeebe-full-helm
+ * zeebe-cluster-helm
+ * zeebe-operate-helm
+ * zeebe-tasklist-helm
+
+The new `ccsm-helm` chart is a full replacement of `zeebe-full-helm` and replaces (contains) all other charts as sub-charts.
```

Comment:
💭 While reading this I remembered that the documentation needs to be updated accordingly as well. (https://docs.camunda.io/docs/self-managed/zeebe-deployment/kubernetes/helm/installing-helm/) I am happy to help out here as well

Reply:
Yes totally agree with this. I have it on my agenda https://github.com/camunda-community-hub/camunda-cloud-helm/issues/123 but would be really happy if I get some support on this :+1: